### DISCUSSION TITLE
Formatting icon

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#1822] [Request: Icon] [answered]
Hi Team,

Can I please request a Formatting 'A' with an underline icon? We are using it as part of the Email HTML Formatting toolbar inside Email in Flex. The icon will appear when you are in the [slim view version of Flex](https://twilio.invisionapp.com/console/share/MH2W33QZ5B/724706744) or when a user hits a certain breakpoint when shrinking the Flex canvas. 

This icon will sit beside the Toolbar icons the Paste team have recently designed.

Please see the examples attached in situ. 

<img width="319" alt="Screenshot 2021-08-31 at 17 15 04" src="https://user-images.githubusercontent.com/49287701/131539115-8cafd49d-83b5-4136-a8ee-73d660d063ac.png">

<img width="706" alt="Screenshot 2021-08-31 at 17 14 53" src="https://user-images.githubusercontent.com/49287701/131539120-9cf3c145-46a5-40d6-9e4c-16a522ab3082.png">

<img width="726" alt="Screenshot 2021-08-31 at 17 14 26" src="https://user-images.githubusercontent.com/49287701/131539123-9d101ac2-65f0-4b5b-bfe6-919daf70adf3.png">

Please let me know if you have any questions. 

Many thanks,
Porlin

**Comment by gloriliale** [comment]
Hi @Porlin! We hear and just made some tickets for adding this icon! We'll respond here when it's released and ready for your use. Thanks!

**Comment by loreina** [comment, marked as answer]
Hey @Porlin, thanks for your patience. The TextFormat icon is now available in Figma as `other/Toolbar/TextFormat` and in code as `TextFormatIcon`. 🎉
